### DISCUSSION TITLE
Differences between self hosting and paid online SocioBoard?

### DISCUSSION THREAD

**Discussion by smokinjo** [socioboard/Socioboard-5.0#325] [General]
I have been looking at SocioBoard with some interest as of late.   It seems to fit our needs.

They mention open source and they there is a version that you can install yourself.

I've been trying to find a list of features that shows what is and is not avaialble in each version(self hosted, vs th eother 2-3 levels of paid accounts).

Can anyone point me to this information?

Thanks!!

**Comment by sureshbabu1995** [comment]
Hi @smokinjo 

If you want to know our application and the features of it please visit our main site  - [SocioBoard](https://socioboard.com/)

And Try create an account and explore all the features in the dashboard.. and sooner we'll be bringing major updates to UI and Features.

**Reply by smokinjo** [reply]
Hello @sureshbabu1995 

If you look at my question, I was not asking for the site you sign up at: socioboard.com

I ask for a comparison of the options available:
1 - online socioboard.com - with different levels of accounts - standard, topaz and ruby
2 - the 3 levels of paid software(I saw a page with $1000, $2000 and $3000, on the wiki I think)
3 - self hosted

Does the self hosted have the same features as the ruby account or the other paid version?   Can we create teams?   In our installed version, there is no admin tools, so we can not make teams yet, that I know of.

Thanks

**Reply by KPKoch** [reply]
I second the question. Besides non working links on the .org page, the information given is sparse.
What concerns me are the connectors aka are there differences in the supported social media channels or are they planned?
At the current status of information I cannot decide if just a crippled core is opensource or if it is even. So ATM I assume the first.

And no, I don't want to have a look at at the GUI of one version, to count the features... even doing this tedious work, I would need a comparison.
